crates/matrix-sdk/src/room/mod.rs

```diff
@@ -315,6 +316,24 @@ impl Room {
         self.client.subscribe_to_room_updates(self.room_id())
     }
 
+    /// Subscribe to typing notifications for this room.
+    ///
+    /// The returned receiver will receive a new vector of user IDs for each
+    /// sync response that contains 'm.typing' event.
+    pub fn subscribe_to_typing_notifications(
+        &self,
+    ) -> (EventHandlerDropGuard, broadcast::Receiver<Vec<OwnedUserId>>) {
+        let (sender, receiver) = broadcast::channel(16);
+        let typing_event_handler_handle = self.client.add_room_event_handler(self.room_id(), {
+            let sender = sender.clone();
+            move |ev: SyncTypingEvent| async move {
+                let _ = sender.send(ev.content.user_ids);
```

Comment:
Please explain why you skip the `Result`.

Reply:
This is a pattern I've seen a lot in the codebase, we just ignore because it should fail only when there is no subscriber, right?

Reply:
Indeed. Let's keep it like that.